### DISCUSSION TITLE
patch command in Nuxt project

### DISCUSSION THREAD

**Discussion by HugeLetters** [pnpm/pnpm#6888] [Q&A]
Could someone point out to me why possibly `patch` command wouldn't work in my project? `.patch` file gets generated with all my changes, `package.json` and lock-file get updated, and even a patched package folder is generated in `node_modules` but it doesn't contain any changes.
https://github.com/openworld-community/ows-events-landings

I thought maybe `shamefully_hoist` is the reason but doesn't seem so. I've tried bootstrapping a new Nuxt project from their template which should have the same set-up as in my repo but there it works fine and can't figure out what's wrong with my set-up.

**Comment by await-ovo** [comment]
Just tried `pnpm patch defu` in your repo, looks everything worked fine:

![image](https://github.com/pnpm/pnpm/assets/41503212/ed3afdad-aad9-45eb-a3ec-367add193c0b)

Can you see `patches` folder generated in project directory on your side ？

**Reply by HugeLetters** [reply]
Yeah, as I've said - correct `.patch` file gets generated. It's that these changes do not get applied to my `node_modules` - I get a new patched module but it doesn't contain my changes.

![image](https://github.com/pnpm/pnpm/assets/119697239/3ccdb0ab-3aab-470a-9e37-92403e924b60)
![image](https://github.com/pnpm/pnpm/assets/119697239/2c606e56-fb35-4c87-bda2-11cb4d4ad4bc)

**Reply by await-ovo** [reply]
Not sure if it has something to do with the diff content, in your patch file it shows `renamed dist/defu.cjs to dist/defu.cjs`.

 I don't know why our patch file looks different if we just change the file.

**Reply by HugeLetters** [reply]
If there's any info I could provide about my system/configuration which might help figure this out - please let me know. 
I'll try to experiment further for now.
